### PR TITLE
chore(billing): Update docs links for managing quotas

### PR DESCRIPTION
**Pull request by isabellaenriquez**
update the quota docs links for transactions and spans :)

### CODE REVIEW

File: static/app/views/settings/account/notifications/utils.tsx [hunk 1]
```diff
@@ -40,9 +40,9 @@ export function getDocsLinkForEventType(
 ) {
   switch (event) {
     case 'transaction':
-      return 'https://docs.sentry.io/product/performance/transaction-summary/#what-is-a-transaction';
+      return 'https://sentry-docs-git-move-quota-to-billing.sentry.dev/pricing/quotas/manage-transaction-quota/';
```
Comment (schew2381) [inline static/app/views/settings/account/notifications/utils.tsx:43]: why is this link so strange? I tested it and it works but still

Comment (schew2381) [reply static/app/views/settings/account/notifications/utils.tsx:43]: weird cuz then going anywhere else on docs also uses that domain

Comment (scttcper) [reply static/app/views/settings/account/notifications/utils.tsx:43]: this is the domain we're putting am2 docs on for now

**Review by scttcper — APPROVED** [review]

File: static/app/views/settings/account/notifications/utils.tsx (hunk 1 above)
Comment (schew2381) [reply static/app/views/settings/account/notifications/utils.tsx:43]: ah ok makes sense

Comment (isabellaenriquez) [reply static/app/views/settings/account/notifications/utils.tsx:43]: i'll double check with the team but this is specifically for billing with transactions (old) vs billing with spans (new)

**Pull request merged**